### PR TITLE
add js code to main page

### PR DESCRIPTION
**Pull request by AnnaIstochnikova**
- add html of main.div
- fetch first 20 films from API

### CODE REVIEW

**Review by krystianklimczak — COMMENTED** [review]
Hej dziś, już nie zdążę Ci sprawdzić poprawności kodu, narazie tylko rozwiązałem konflikt, jutro zasiąde na spokojnie do kodu i na pewno się do czegoś "doczepię" 💪

File: src/js/filmApi.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ export const responseAxios = axios
   .catch(error => console.log(error));
 
 // SECOND METHOD
-const options = {
+export const options = {
```
Comment (krystianklimczak) [inline src/js/filmApi.js:23]: delete this export, fn gonna be reuseable for more users

File: src/js/main.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+            <p class="main__photo--vote">
+              ${film.vote_average}
+            </p>
+          </div>
+        </div>`;
+    mainContainer.insertAdjacentHTML('beforeend', filmBox);
+  });
+}
+
+makeFilmsBox();
```
Comment (krystianklimczak) [inline src/js/main.js:86]: call this makeFilmsBox() fn in our index.js. Don't forget to export/import it from your code to let it be reuseable for more features

File: src/js/main.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+            </p>
+            <p class="main__photo--genre">
+              <b>${genreList}</b>
+            </p>
+            <p class="main__photo--year">
+              <b>${film.release_date.substring(0, 4)}</b>
+            </p>
+            <p class="main__photo--vote">
+              ${film.vote_average}
+            </p>
```
Comment (krystianklimczak) [inline src/js/main.js:79]: we don't need rating on our home posters but if you think that is good idea let us know about it at meeting.

Comment (krystianklimczak) [reply src/js/main.js:79]: Implement one more feature where we gonna have button with link to movie trailer as we talk about it at meeting

**Review by krystianklimczak — COMMENTED** [review]
Checkout changes.

**Comment by krystianklimczak** [comment]
Dear Ania I.
I review your code, check it out and make changes.
Wish you have a good weekend and see you ✌️

File: src/js/main.js (hunk 3 above)
Comment (AnnaIstochnikova) [reply src/js/main.js:79]: Regarding the rating, yes, my mistake. We will need it in 'My Library'. We will have a button, which I think is enough.

File: src/js/filmApi.js [hunk 4]
```diff
@@ -33,8 +35,24 @@ export const axiosSecondFetchFn = async () => {
...
+//FETCHING ALL GENRES LIST
+
+export const fetchGenres = async () => {
+  try {
+    const response = await axios.get('https://api.themoviedb.org/3/genre/movie/list', {
+      params: {
+        api_key: API_KEY,
+      },
+    });
+    const genres = response.data.genres;
```
Comment (krystianklimczak) [inline src/js/filmApi.js:53]: You forget about "await" =>` const genres = await response.data.genres;`

File: src/js/main.js [hunk 5]
```diff
@@ -60,27 +32,22 @@ function drawFilmBox(films) {
       });
     }
 
-    const filmBox = `<div class="main__photo--card data-id="${film.id}" ">
```
Comment (krystianklimczak) [inline src/js/main.js]: quotes => `div class="main__photo-card" data-id="${film.id}"`

**Review by krystianklimczak — COMMENTED** [review]
For html structure use BEM convention
1. main-section
2. film-poster
3. film-poster__photo
4. film-poster__description
5. film-poster__title
6. film-poster__button
7. film-poster__genre
8. film-poster__year
Let's say that our main-section is block and our poster are separeted block's that have their own element. Keep it like this

**Review by krystianklimczak — APPROVED** [review]
1.export fn drawFilmBox(films) to make it reuseable
2. when you using fetchGenres() push actual data to your empty array with ... (rest) because we need array with all genres => `allGenres.push(...data)`
3. add condition to drawFilmBox fn that let us check about pages to clear actual state of our site

**Review by krystianklimczak — COMMENTED** [review]
1.export fn drawFilmBox(films) to make it reuseable
2. when you using fetchGenres() push actual data to your empty array with ... (rest) because we need array with all genres => allGenres.push(...data)
3. add condition to drawFilmBox fn that let us check about pages to clear actual state of our site

**Review by annienf17 — APPROVED** [review]

**Review by annienf17 — APPROVED** [review]

**Review by krystianklimczak — APPROVED** [review]
Great job 💪

**Pull request merged**